docs/Cryptography/index.md

```diff
@@ -0,0 +1,21 @@
+# Cryptography
+
+A variety of cryptography functions will be added to Community Framework in the future. 
+
+## SHA-256
+
+```csharp
+string path = "$profile:test.txt";
+
+// Create the file
+CF_TextReader reader = new CF_TextReader(new CF_FileStream(path, FileMode.READ));
+Print(reader.ReadLine()); // "This is some text"
+
+CF_Base16Stream hexStream = new CF_Base16Stream();
```

Comment:
rename hexStream to `output`

That makes it easier to understand what is what in the `Process` call